packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -2818,6 +2818,7 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     RenderBox child,
     bool container: false,
     bool explicitChildNodes,
+    bool disabled,
```

Comment:
Here and elsewhere: unless there's a really good reason to do so, prefer positive bool names  (`enabled` in this case). The example above (`disabled: !enabled`) is a decent example of why.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-double-negatives-in-apis

Reply:
+1